docs/upgrade.html

```diff
@@ -19,6 +19,16 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_9_0" href="#upgrade_3_9_0">Upgrading to 3.9.0 from any version 0.8.x through 3.8.x</a></h4>
```

Comment:
one question: this issue is existent in 3.8.0 also. maybe we should write it for 3.8.0?

Reply:
I will do this in the 3.8 branch. Same with 3.7.
But I think right now as there is no 3.8.1 in the making nor 3.7.2 might be misleading. Any thoughts?

Reply:
> But I think right now as there is no 3.8.1 in the making nor 3.7.2 might be misleading. Any thoughts?

Maybe we can remind readers that is a known issue in 3.8.0?

Reply:
Done